webapp/channels/src/components/file_preview_modal/image_preview.tsx

```diff
@@ -42,6 +53,7 @@ export default function ImagePreview({fileInfo, canDownloadFiles}: Props) {
                 data-testid='imagePreview'
                 alt={'preview url image'}
                 src={previewUrl}
+                {...conditionalSVGStyleAttribute}
```

Comment:
In the other component, you passed this in using the `style` prop. Should we do that here as well?

Reply:
Oops, I thought I did that, thank you.